concepticondata/conceptlists/Zhou-2020-350.tsv

```diff
@@ -0,0 +1,351 @@
+ID	NUMBER	ENGLISH	CHINESE	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Zhou-2020-350-1	1	all	全部	98	ALL
+Zhou-2020-350-2	2	almost	快要		
+Zhou-2020-350-3	3	ant	螞蟻	587	ANT
+Zhou-2020-350-4	4	ashes	灰	646	ASH
+Zhou-2020-350-5	5	at first	先	2915	AT FIRST
+Zhou-2020-350-6	6	bad	壞	1292	BAD
+Zhou-2020-350-7	7	bamboo	竹子	1927	BAMBOO
+Zhou-2020-350-8	8	bamboo mat	竹蓆	3240	BAMBOO MAT
+Zhou-2020-350-9	9	beast	獸	958	WILD ANIMAL
+Zhou-2020-350-10	10	bed	牀	1663	BED
+Zhou-2020-350-11	11	belly	肚子	1251	BELLY
+Zhou-2020-350-12	12	belt	腰帶	772	BELT
+Zhou-2020-350-13	13	big	大	1202	BIG
+Zhou-2020-350-14	14	bitter	苦	887	BITTER
+Zhou-2020-350-15	15	black	黑	163	BLACK
+Zhou-2020-350-16	16	blood	血	946	BLOOD
+Zhou-2020-350-17	17	boat	船	1844	BOAT
+Zhou-2020-350-18	18	boiling water	開（水）	3362	BOILED WATER
+Zhou-2020-350-19	19	bottom	底下	834	BUTTOCKS
+Zhou-2020-350-20	20	bovine	牛	3044	BOVINE
+Zhou-2020-350-21	21	bowl	碗	1530	BOWL
+Zhou-2020-350-22	22	bran	糠	2362	BRAN
+Zhou-2020-350-23	23	bright	亮	679	BRIGHT
+Zhou-2020-350-24	24	bug	蟲	1027	BUG
+Zhou-2020-350-25	25	buy	買	1869	BUY
+Zhou-2020-350-26	26	cat	貓	1208	CAT
+Zhou-2020-350-27	27	cave	洞	1155	CAVE
+Zhou-2020-350-28	28	cedar tree	杉樹		
+Zhou-2020-350-29	29	chilly	涼		
```

Comment:
We mapped the Chinese gloss to COLD (OF WEATHER) before so it should be mapped here as well.

Reply:
yes!